.flake8

```diff
@@ -1,5 +1,66 @@
 [flake8]
-ignore = E203, E266, E501, W503, W605, N812
-exclude = .git,docs
 max-line-length = 119
-
+ignore =
+    # Unnecessary dict call - rewrite as a literal.
+    C408
+    # Missing docstring in public class
+    D101
+    # Missing docstring in public method
+    D102
+    # Too many leading '#' for block comment
+    E266
```

Comment:
@martinkim0 I added these but we should remove them and add noqa's instead of documenting everything